src/pyfmi/fmi3.pyx

```diff
@@ -880,6 +880,17 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
         rtol = self.get_relative_tolerance()
         return 0.01*rtol*self.nominal_continuous_states
 
+    cpdef get_variable_unbounded(self, variable_name):
+        """TODO:"""
+        return False
+
+    def get_generation_tool(self):
```

Comment:
Thought with that one I could get a proper exception for testing sensitivities, but would needs a few more functions. 
Anyways, might as well keep it.